src/xlsx.rs

```diff
@@ -409,6 +379,37 @@ fn get_row_column(range: &[u8]) -> Result<(u32, u32)> {
     Ok((row - 1, col - 1))
 }
 
+/// attempts to read either a simple or richtext string
+fn read_string(xml: &mut Reader<BufReader<ZipFile>>, closing: &[u8]) -> Result<Option<String>> {
+    let mut buf = Vec::new();
+    let mut rich_buffer: Option<String> = None;
+    loop {
+        match xml.read_event(&mut buf) {
+            Ok(Event::Start(ref e)) if e.name() == b"r" => {
+                if rich_buffer.is_none() {
+                    // use a buffer since richtext has multiples <r> and <t> for the same cell
+                    rich_buffer = Some(String::new());
+                }
+            }
+            Ok(Event::End(ref e)) if e.name() == closing => {
+                return Ok(rich_buffer);
+            }
+            Ok(Event::Start(ref e)) if e.name() == b"t" => {
+                let value = xml.read_text(b"t", &mut Vec::new())?;
+                if let Some(ref mut s) = rich_buffer {
+                    s.push_str(&value);
+                } else {
+                    return Ok(Some(value));
```

Comment:
Perhaps we should call [`xml.read_to_end(closing, &mut Vec::new())?`](https://docs.rs/quick-xml/0.6.2/quick_xml/reader/struct.Reader.html#method.read_to_end) first ?